src/sftp.h

```diff
@@ -153,9 +153,10 @@ struct _LIBSSH2_SFTP
     uint32_t last_errno;
 
     /* Holder for partial packet, use in libssh2_sftp_packet_read() */
-    unsigned char partial_size[4];      /* buffer for size field   */
-    size_t partial_size_len;            /* size field length       */
-    unsigned char *partial_packet;      /* The data                */
+    unsigned char packet_header[9];
+    /*packet size (4) packet type (1) request id (4) */
```

Comment:
```suggestion
    /* packet size (4) packet type (1) request id (4) */
```
make it pass checksrc.